cluster/addons/dashboard/dashboard-rbac.yaml

```diff
@@ -7,10 +7,6 @@ metadata:
   name: kubernetes-dashboard-minimal
   namespace: kube-system
 rules:
-  # Allow Dashboard to create 'kubernetes-dashboard-key-holder' secret.
```

Comment:
+1 for this change, probably need to add an empty EnsureExists `kubernetes-dashboard-key-holder` Secret to the secrets.yaml file